### PR TITLE
lesson09 assignment09 mailroom_oo

### PR DESCRIPTION
**Pull request by mmanc125uw**

### CODE REVIEW

File: students/mmancini/lesson09/donor_models.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class Donor():
+    def __init__(self, full_name):
+        self.name = full_name
+        self.lst_donations = []
+
+    def __init__(self, full_name, in_donations):
+        self.name = full_name
+        self.lst_donations = in_donations
```
Comment (natasha-aleksandrova) [inline students/mmancini/lesson09/donor_models.py:14]: two init methods? (this isn't a correct implementation)

File: students/mmancini/lesson09/donor_models.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+
+###################################
+
+
+class Donor():
```
Comment (natasha-aleksandrova) [inline students/mmancini/lesson09/donor_models.py:7]: the class needs more properties / methods

File: students/mmancini/lesson09/donor_models.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class DonorCollection():
+    def __init__(self, donors=None):
+        if donors is None:
+            donors = {}
+        self.dict_donors = donors
+
+    def add_donor(self, in_donor):
+        self.dict_donors[in_donor.name] = in_donor.lst_donations
```
Comment (natasha-aleksandrova) [inline students/mmancini/lesson09/donor_models.py:24]: this class should hold reference to Donor objects - however does not appear to be the case in this class.

**Comment by mmanc125uw** [comment]
Hi Natasha
   I submitted another pull request to address your comments, specifically adding property and methods to Donor class, and class DonorCollection now contains Donor objects.

https://github.com/UWPCE-PythonCert-ClassRepos/SP_Online_PY210/pull/869


Thanks
Mike





________________________________

**Pull request merged**